examples/waitset_status_cond/c++11/USER_QOS_PROFILES.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0"?>
+<!--
+ (c) 2005-2017 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+ RTI grants Licensee a license to use, modify, compile, and create derivative
+ works of the Software.  Licensee has the right to distribute object form only
+ for use with RTI products.  The Software is provided "as is", with no warranty
+ of any type, including any warranty for fitness for any purpose. RTI is under
+ no obligation to maintain or support the Software.  RTI shall not be liable for
+ any incidental or consequential damages arising out of the use or inability to
+ use the software.
+ -->
+<!-- Description XML QoS Profile for waitsets The QoS configuration of the
+    DDS entities in the generated example is loaded from this file. This file
+    is used only when it is in the current working directory or when the enviroment
+    variable NDDS_QOS_PROFILES is defined and points to this file. For more information
+    about XML QoS Profiles see Chapter 15 in the RTI Connext user manual. -->
+
+<dds xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+     xsi:noNamespaceSchemaLocation="C:/RTI/rti-connext-dds/rti_connext_dds-5.3.0/bin/../resource/schema/rti_dds_qos_profiles.xsd"
```

Comment:
Change the URI for the XSD with the online version: https://community.rti.com/schema/5.3.0/rti_dds_qos_profiles.xsd